Main/Source/nonhuman.cpp

```diff
@@ -960,9 +960,10 @@ truth vampirebat::SpecialBiteEffect(character* Victim, v2 HitPos, int BodyPartIn
 }
 
 bool ChamaleonPolymorphRandomly(chameleon* c){
-  if(!c->StateIsActivated(POLYMORPH_LOCK)){
```

Comment:
Did you accidentally nerf this feature (I think it allows a chameleon to wear a ring of unchanging(?) and remain in that form). Or is this inherently a bug-fix?

Reply:
That's already checked in `character::PolymorphRandomly()` https://github.com/Attnam/ivan/blob/1cb0d19f79117832ca1d099af28785238216f3f5/Main/Source/char.cpp#L6601-L6610 unless we want to optimize it a bit or be more explicit.

Reply:
Ahh I see, I should've seen that earlier. Great!